arch/xtensa/core/xtensa_intr.c

```diff
@@ -14,14 +14,14 @@
 #include "xtensa_rtos.h"
 #include "xtensa_api.h"
 #include <kernel_structs.h>
+#include <sw_isr_table.h>
 
 #if XCHAL_HAVE_EXCEPTIONS
 
 /* Handler table is in xtensa_intr_asm.S */
 
 extern xt_exc_handler _xt_exception_table[XCHAL_EXCCAUSE_NUM];
```

Comment:
we should get rid of all occurrences of this and replace with the actual table, its just an alias and is legacy Cadence cruft